cli/tests/integrations/test_package_bundle.py

```diff
@@ -60,6 +60,18 @@ def test_bundle_fail_missing_package_json():
     assert not glob.glob(_PACKAGE_NAME_GLOB)
 
 
+def test_bundle_fail_duplicate_asset_names():
+    common.assert_command(
+        ['dcos', 'package', 'bundle', '--output-directory=/tmp',
+         'tests/data/package/invalid-asset-names-directory'],
+        returncode=1,
+        stderr=(b'Error bundling package. Multiple assets have the '
+                b'same filename [\'cassandra-mesos.gz\', '
+                b'\'cassandra-mesos.tar.gz\']\n'))
+
+    assert not glob.glob(_PACKAGE_NAME_GLOB)
```

Comment:
We need to change the test so that the file are `cassandra-mesos.tar.gz` and `cassandra-mesos-tar-gz`.